### PR TITLE
Update contribute to translations

### PR DESCRIPTION
**Pull request by gforcada**
Changes proposed in this pull request:

- removed references to older Plone versions
- use semantic new lines
- fix outdated references (plone5 key on i18n.cfg for example)
- make it, more or less, version agnostic

### CODE REVIEW

**Comment by gforcada** [comment]
@vincentfretin would you mind having a look at it?

File: develop/plone/i18n/contribute_to_translations.rst [hunk 1]
```diff
@@ -55,23 +45,21 @@ Here you have the following directories:
...
 - ``locales`` used for core Plone translations.
 - ``locales-addons`` used for some addons packages.
-- ``locales-future`` used for Plone 5 packages. The po files can change
-  really often. The translations will normally be merged to the locales
-  directory when Plone 5 will be released. This directory is not included in
-  ``plone.app.locales`` 4.x releases. For developers: if you want to add a
-  Plone 5 package to this directory, you can add it to the ``plone5``
-  variable in ``experimental/i18n.cfg``, don't forget to add a line in
-  ``[sources]`` if it's missing.
+- ``locales-future`` used for Plone 5 packages.
```
Comment (vincentfretin) [inline develop/plone/i18n/contribute_to_translations.rst]: I thought I fixed this documentation already, maybe the plone5 version of the documentation is in another branch? locales-future doesn't exist anymore on Plone 5, you can remove this bullet point.

**Comment by vincentfretin** [comment]
Other than removing the bullet point locales-future, LGTM.

**Comment by vincentfretin** [comment]
Yeah in branch 5.0, this is more up to date, see https://github.com/plone/documentation/blob/5.0/develop/plone/i18n/contribute_to_translations.rst

**Comment by gforcada** [comment]
@vincentfretin I removed the bullet point. @plone/documentation-team if @vincentfretin does not have more comments, I would ❤️ to hear yours, or have the branch merged 😄

File: develop/plone/i18n/contribute_to_translations.rst [hunk 2]
```diff
@@ -10,27 +10,17 @@ How to contribute to Plone core translations
 Introduction
 ============
 
-You need to have write access to
-https://github.com/collective/plone.app.locales to be able to commit
-your translation directly.
+You need to have write access to https://github.com/collective/plone.app.locales to be able to commit your translation directly.
```
Comment (jensens) [inline develop/plone/i18n/contribute_to_translations.rst]: First, this is, in fact, write access to the collective. Which is explained here https://collective.github.io/

Also, this is not true. 
One may fork the repository and create a PR as explained below.

**Comment by gforcada** [comment]
@jensens thanks for the feedback, I updated it. Any more comments on it?

File: develop/plone/i18n/contribute_to_translations.rst [hunk 3]
```diff
@@ -87,24 +72,24 @@ You can commit your translation from this locales directory:
...
-Press button *Create Pull request*. Fill it in.
+- Go to GitHub ``https://github.com/YOURUSERNAME/plone.app.locales``
+- Press button *Create Pull request*.
+  Fill it in.
 
 The request will appear for *plone.app.locales* authors.
-If it does not get merged in timely manner, poke people on the #plone IRC channel
-or the mailing list below (sometimes requests go unnoticed).
+If it does not get merged in timely manner,
+ask on the #plone IRC channel or the mailing list below (sometimes requests go unnoticed).
```
Comment (svx) [inline develop/plone/i18n/contribute_to_translations.rst]: Should we change that to community.plone.org and gitter ?

Comment (gforcada) [reply develop/plone/i18n/contribute_to_translations.rst]: Or rather, should we point to a generic page https://plone.org/support ? Or is there any support-like page already in the documentation?

Comment (gforcada) [reply develop/plone/i18n/contribute_to_translations.rst]: Still, I would make a decision on that and then do a search and replace on all the docs that point to IRC to point to this overview page. Does it make sense?

**Comment by svx** [comment]
IMHO in this case pointing someone to a generic page is not that cool :)

For general questions, yes - but in this case here it is about a certain audience so IMHO we should point to community even to a i18n sub section if we have one :)

@polyester any opinions ?

**Comment by polyester** [comment]
We do have a i18n section of community at https://community.plone.org/c/development/i18nl10n, but it is a bit empty. However, that's a chicken-and-egg question, so maybe just point to it, at least they will be at community.plone.org then. 

and point to gitter

**Comment by gforcada** [comment]
@plone/documentation-team I made yet another update, would you mind having a look and merging? 😄

File: develop/plone/i18n/contribute_to_translations.rst [hunk 4]
```diff
@@ -10,27 +10,20 @@ How to contribute to Plone core translations
 Introduction
 ============
 
-You need to have write access to
-https://github.com/collective/plone.app.locales to be able to commit
-your translation directly.
-You can also update a po file online and make a pull request.
+You might want to have write access to https://github.com/collective/plone.app.locales to be able to commit your translation directly.
+For that, `join the collective github organization <https://collective.github.io/>`_.
```
Comment (svx) [inline develop/plone/i18n/contribute_to_translations.rst]: It should be GitHub, not github :)

File: develop/plone/i18n/contribute_to_translations.rst [hunk 5]
```diff
@@ -87,27 +72,28 @@ You can commit your translation from this locales directory:
...
 you can do the following:
 
-Login to GitHub. Go to GitHub `plone.app.locales <https://github.com/collective/plone.app.locales>`_
-
-Press *Fork*. Now GitHub creates a copy of ``plone.app.locales`` package for you.
-
-Then on your computer in ``plone.app.locales`` do a special git push to your own repository::
+- Login to GitHub.
+- Go to GitHub `plone.app.locales <https://github.com/collective/plone.app.locales>`_
+- Press *Fork*.
```
Comment (svx) [inline develop/plone/i18n/contribute_to_translations.rst]: In place of that we shoud use :guilabel: :)

File: develop/plone/i18n/contribute_to_translations.rst [hunk 6]
```diff
@@ -87,27 +72,28 @@ You can commit your translation from this locales directory:
...
+  Now GitHub creates a copy of ``plone.app.locales`` package for you.
+- Then on your computer in ``plone.app.locales`` do a special git push to your own repository::
 
     git push git@github.com:YOURUSERNAMEHERE/plone.app.locales.git
 
-Go to GitHub ``https://github.com/YOURUSERNAME/plone.app.locales``
-
-Press button *Create Pull request*. Fill it in.
```
Comment (svx) [inline develop/plone/i18n/contribute_to_translations.rst:103]: See above

File: develop/plone/i18n/contribute_to_translations.rst [hunk 7]
```diff
@@ -87,27 +72,28 @@ You can commit your translation from this locales directory:
...
+- Go to GitHub ``https://github.com/YOURUSERNAME/plone.app.locales``
+- Press button *Create Pull request*.
```
Comment (svx) [inline develop/plone/i18n/contribute_to_translations.rst]: Same here :)

**Comment by svx** [comment]
@gforcada I am on it

**Comment by gforcada** [comment]
@svx oh cool, thanks!

**Pull request merged**